### ISSUE TITLE
AttributeError: 'Connection' object has no attribute 'message'

### ISSUE THREAD

**Issue by absorber** [JonnyHaystack/i3-resurrect#26]
Hi,

I can invoke `i3-resurrect` just fine:

```
└─$   i3-resurrect
Usage: i3-resurrect [OPTIONS] COMMAND [ARGS]...

Options:
  --version   Show the version and exit.
  -h, --help  Show this message and exit.

Commands:
  restore  Restore i3 workspace layout and programs.
  save     Save an i3 workspace's layout and running programs to a file.
```

...but when I try to save a workspace, it prints the following error:

```
└─$  i3-resurrect save -w 6
Traceback (most recent call last):
  File "/home/me/.local/bin/i3-resurrect", line 11, in <module>
    sys.exit(main())
  File "/home/me/.local/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/me/.local/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/me/.local/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/me/.local/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/me/.local/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/me/.local/lib/python3.6/site-packages/i3_resurrect/main.py", line 54, in save_workspace
    save_layout(workspace, directory, swallow_criteria)
  File "/home/me/.local/lib/python3.6/site-packages/i3_resurrect/main.py", line 67, in save_layout
    workspace_tree = util.get_workspace_tree(workspace)
  File "/home/me/.local/lib/python3.6/site-packages/i3_resurrect/util.py", line 88, in get_workspace_tree
    root = json.loads(i3.message(i3ipc.MessageType.GET_TREE, ''))
AttributeError: 'Connection' object has no attribute 'message'
```

I have `~/.local/bin` in my `$PATH` var.

Since I'm on Ubuntu 16.04, which only has Python 3.5, I've installed Python 3.6 using `pyenv` according to [this AskUbuntu answer](https://askubuntu.com/a/865644).

**Comment by zefreak** [comment]
I am getting this same error on arch linux after installing through aur

**Comment by JonnyHaystack** [comment]
Hm, this is strange. Sorry I'm short on spare time at the moment but I'll get on this as soon as I have the chance.

Meanwhile, please provide the output of `i3-resurrect --version` and `i3 --version`

Also does the load workspace command give the same error?

**Comment by zefreak** [comment]
i3-resurrect --version gives version 1.3.1
i3 --version gives version 4.17

**Comment by JonnyHaystack** [comment]
Should be fixed in #27. I've released this in 1.3.2. Let me know if it's working for you now.
